salt/tops/ext_nodes.py

```diff
@@ -74,15 +74,18 @@ def top(**kwargs):
     """
     if "id" not in kwargs["opts"]:
         return {}
-    proc = subprocess.run(
+    _cmd_str = " ".join(
```

Comment:
I would prefer to fix this so we are not using shell=True and we pass in a list to subprocess.run.

Reply:
Thanks for your review! I removed the parameter and adapted the tests.